rwengine/include/rw_mingw.hpp

```diff
@@ -0,0 +1,3 @@
+#undef IGNORE
+#undef near
+#undef far
```

Comment:
Those sound required. The windows headers should definitely be isolated from the rest of the code eventually, but this header is an OK fix for now.